packages/twenty-server/src/modules/reports/standard-objects/analytics-query-filter.workspace-entity.ts

```diff
@@ -0,0 +1,65 @@
+import { Relation } from 'typeorm';
+
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+import { ANALYTICS_QUERY_FILTER_STANDARD_FIELD_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids';
+import { STANDARD_OBJECT_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-object-ids';
+import { WorkspaceEntity } from 'src/engine/twenty-orm/decorators/workspace-entity.decorator';
+import { WorkspaceField } from 'src/engine/twenty-orm/decorators/workspace-field.decorator';
+import { WorkspaceIsNullable } from 'src/engine/twenty-orm/decorators/workspace-is-nullable.decorator';
+import { BaseWorkspaceEntity } from 'src/engine/twenty-orm/base.workspace-entity';
+import { RelationMetadataType } from 'src/engine/metadata-modules/relation-metadata/relation-metadata.entity';
+import { WorkspaceRelation } from 'src/engine/twenty-orm/decorators/workspace-relation.decorator';
+import { AnalyticsQueryWorkspaceEntity } from 'src/modules/reports/standard-objects/analytics-query.workspace-entity';
+import { WorkspaceJoinColumn } from 'src/engine/twenty-orm/decorators/workspace-join-column.decorator';
+
+@WorkspaceEntity({
+  standardId: STANDARD_OBJECT_IDS.analyticsQueryFilter,
```

Comment:
I wonder how much of this is the same as viewFilter and should be 100% shared? let's keep them as close as possible?

Reply:
I think this also depends on whether we want to be able to create JOIN queries or not. If not, I think we can use view-filter, otherwise a separate entity is required.